pttrack/templates/pttrack/base.html

```diff
@@ -38,6 +40,7 @@
     body {
       padding-top: 50px;
       padding-bottom: 5%;
+      background-color: rgb(238, 242, 245)
```

Comment:
Why the addition of a background color here?

Reply:
I've played around a little with the design, this can be deleted and slipped into the commit by mistake